build.sbt

```
@@ -2,7 +2,7 @@ lazy val root = (project in file("."))
   .settings(
     name := "scala-http-client",
     organization := "io.moia",
-    version := "1.1.0-akka2.5",
+    version := "1.1.0",
```

Hm, we'll need versions for both 2.5 and 2.6. I'm also not a fan of the suffix, but I don't know a better way to differentiate between them. With this change, we can't anymore, right?

AND: We should increment the version number and either release it or add `-SNAPSHOT` 😊 
```suggestion
    version := "1.2.0-SNAPSHOT",
```